### PR TITLE
add ansible and chef

### PR DESCRIPTION
**Pull request by milesstoetzner**

### CODE REVIEW

File: artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Ansible/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (lharzenetter) [inline artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Ansible/LICENSE]: Update this: 2022 University of Stuttgart

File: artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Chef/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (lharzenetter) [inline artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Chef/LICENSE]: s.a.

**Review by lharzenetter — CHANGES_REQUESTED** [review]

File: artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Ansible/LICENSE (hunk 1 above)
Comment (milesstoetzner) [reply artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Ansible/LICENSE]: done

File: artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Chef/LICENSE (hunk 2 above)
Comment (milesstoetzner) [reply artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/Chef/LICENSE]: done

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**